go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/giantswarm/certs v0.2.0
 	github.com/giantswarm/errors v0.2.3
 	github.com/giantswarm/k8sclient v0.2.0
-	github.com/giantswarm/k8scloudconfig/v6 v6.4.0
+	github.com/giantswarm/k8scloudconfig/v7 v7.0.0
```

Comment:
Might be good to use v7.0.3 here to get rid of the missed v6 import (https://github.com/giantswarm/k8scloudconfig/blob/master/CHANGELOG.md#701---2020-07-20).